relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -545,7 +540,7 @@ mod tests {
         chrome_extension,
         "chrome-extension://begnopegbbhjeeiganiajffnalhlkkjb/img/assets/icon-10k.svg",
         "resource.other",
-        "chrome-extension://*/img/assets/icon-*k.svg"
+        "browser-extension://*"
```

Comment:
@DominikB2014 my idea here was to group all browser extension resources together so they can be easily filtered out on the querying side. Is this an acceptable solution, or do we need to actually remove those spans from our data?

Reply:
Yeah i think this would be good, if can can filter out all `browser-extensions` when querying.

One question, In this case what would the domain be set to? We've been having very random domains set for chrome extensions
![image](https://github.com/getsentry/relay/assets/44422760/f94a9e23-3b49-4e18-b6c6-592934890182)

Reply:
Good question, let me add a test for that.

Reply:
With the current change, the domain will be `"*"`.

Reply:
So to filter out extensions, we would add `span.domain != '*'`?

Reply:
I guess the safer way would be to filter on the `group` or `description != 'browser-extension://*`, because other resources could theoretically also have `domain == '*'`.

Reply:
Yeah that makes sense, this should work for the product then, if there's any issues (I don't suspect any), we can revist.
Thanks for making the update @jjbayer!